#!/usr/bin/env python
PACKAGE = "vision_demo"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add( "H_min", int_t, 0, "H_min parameter", 38, 0, 255)
gen.add( "H_max", int_t, 0, "H_max parameter", 57, 0, 255)
gen.add( "S_min", int_t, 0, "S_min parameter", 36, 0, 255)
gen.add( "S_max", int_t, 0, "S_max parameter", 208, 0, 255)
gen.add( "V_min", int_t, 0, "V_min parameter", 113, 0, 255)
gen.add( "V_max", int_t, 0, "V_max parameter", 255, 0, 255)

exit(gen.generate(PACKAGE, "color_detection_demo", "HSVColorDetection"))
